readthedocs/templates/projects/projects_search_analytics.html

```diff
@@ -12,6 +12,11 @@
 
 {% block project_edit_content %}
 
+  {% if not enabled %}
+    {% block disabled %}
```

Comment:
We should probably put the same `This beta feature can be enabled by <a href="mailto:{{ SUPPORT_EMAIL }}">asking support</a>.` block here for when its disabled, even if that isn't normally happening. I wonder if this should be in a base template, instead of one-off for each feature.

Reply:
I have moved it to an include :+1: